examples/mdetr/test/test_matcher.py

```diff
@@ -0,0 +1,127 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import random
+
+import pytest
+import torch
+from examples.mdetr.matcher import HungarianMatcher
+from test.test_utils import assert_expected, set_rng_seed
+from torchvision.ops.boxes import box_convert
+
+
+@pytest.fixture(scope="class", autouse=True)
```

Comment:
I suppose here you want to refresh the seed only at class level?
This means the order of the methods making use of random within a class matters. 
The latter can cause some surprise later on, e.g., when someone wants to add a new method with randomness into the same class.